rln/src/public.rs

```diff
@@ -863,6 +904,28 @@ impl RLN {
         Ok(())
     }
 
+    // Generate RLN Proof using a witness calculated from outside zerokit
```

Comment:
Make function description with  /// - we wanna this comments in docs. And add description about input value like in another functions here